app/uk/gov/hmrc/tai/model/templates/PdfSubmission.scala

```diff
@@ -17,18 +17,16 @@
 package uk.gov.hmrc.tai.model.templates
 
 import java.util.UUID
-
-import org.joda.time.LocalDateTime
-import org.joda.time.format.DateTimeFormat
-
+import java.time.{Instant, LocalDateTime, ZoneId}
+import java.time.format.DateTimeFormatter
 import scala.util.Random
 
 case class PdfSubmission(
   customerId: String, //Needs to be the nino without suffix as DMS doesn't use suffix
   formId: String,
   numberOfPages: Int,
   attachmentCount: Int = 0,
-  hmrcReceivedAt: LocalDateTime = LocalDateTime.now(),
+  hmrcReceivedAt: LocalDateTime = LocalDateTime.now(ZoneId.of("Europe/London")), //TODO extract from config
```

Comment:
has a ticket been raised for this? If so can we merge this before that ticket has been done?

Reply:
Yes: PAYEE-255